lib/src/components/NxRadio/__tests__/NxRadio.test.tsx

```diff
@@ -21,146 +22,109 @@ describe('NxRadio', function() {
     children: undefined
   };
 
-  const getShallow = getShallowComponent<Props>(NxRadio, simpleProps);
+  const renderEl = rtlRenderElement(NxRadio, minimalProps);
+  const quickRender = rtlRender(NxRadio, minimalProps);
 
   it('renders a <label> containing a radio <input> and .nx-radio__circle and .nx-radio__outer-circle  elements',
       function() {
-        const shallowRender = getShallow();
-
-        expect(shallowRender).toMatchSelector('label.nx-radio');
+        const el = renderEl()!;
 
-        expect(shallowRender.find('input')).toHaveProp('name', 'color');
-        expect(shallowRender.find('input')).toHaveProp('type', 'radio');
-        expect(shallowRender.find('input')).toHaveProp('disabled', false);
-        expect(shallowRender.find('input')).toHaveProp('checked', false);
+        expect(el.tagName).toBe('LABEL');
 
-        expect(shallowRender.find('input')).toHaveClassName('nx-radio__input');
-
-        expect(shallowRender.find('svg.nx-radio__circle')).toExist();
-        expect(shallowRender.find('.nx-radio__circle .nx-radio__outer-circle')).toExist();
+        expect(quickRender().queryByRole('radio')).toHaveAttribute('name', 'color');
```

Comment:
Sorry, everywhere I said "query by role" I didn't mean the literal `queryByRole` function, I meant any appopriate [RTL query function](https://testing-library.com/docs/queries/about/) by role.  `queryByRole` itself is actually rarely used.  The only difference between it and `getByRole` is that if the element isn't found, `getByRole` will throw an exception (with a very elaborate and helpful error message) while `queryByRole` will just return `null`.  As such, the documented guidance is to always use `getByRole` unless you are trying to prove that a particular element is _not_ present.

Reply:
> Sorry, everywhere I said "query by role" I didn't mean the literal `queryByRole` function, I meant any appopriate [RTL query function](https://testing-library.com/docs/queries/about/) by role. `queryByRole` itself is actually rarely used. The only difference between it and `getByRole` is that if the element isn't found, `getByRole` will throw an exception (with a very elaborate and helpful error message) while `queryByRole` will just return `null`. As such, the documented guidance is to always use `getByRole` unless you are trying to prove that a particular element is _not_ present.

Ohh okay, thanks for explaining the difference. Changed here: 00092cc4c5329310821b561d1677cdbf73832497